paths/electron/starting-with-electron/08-add-to-js.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Add JavaScript
+permalink: /electron/starting-with-electron/add-to-javascript
+next-page: /electron/starting-with-electron/celebrate
+facilitator: false
+sidebar:
+  nav: "electron-getting-starting"
+main-content: |
+
+  Finally, let's add the fun part: the Javascript!
+
+  1. Copy the Javascript from Codepen into the `src/script.js` file.
+  1. Spin up your app, and see how it works.
+  1. Save and commit your changes.
```

Comment:
I assumed at the end of this step that I'd see a fully working app, but I didn't 😢 

![screen shot 2017-08-04 at 5 26 30 pm](https://user-images.githubusercontent.com/16547949/28987642-162063ec-793a-11e7-9d26-f6a3bdb8aec5.png)

And on the command line: 

```bash
2017-08-04 17:26:28.375 Electron Helper[9196:5718091] Couldn't set selectedTextBackgroundColor from default ()
```